src/schema/objects/associated_data.yaml

```diff
@@ -4,21 +4,21 @@
 # Information about whether these directories are required or not is present in
 # `rules/associated_data.yaml`.
 code:
-  name: Code
+  display_name: Code
```

Comment:
@rwblair My disruption of the schema demanded that I create a merge conflict with your PR. I offer my humblest apologies.